node/utils/src/lib.rs

```diff
@@ -66,3 +66,29 @@ where
     let new_struct: S = toml::from_str(toml_string)?;
     Ok(new_struct)
 }
+
+/// Get file count in a certain directory
+/// Will return the number of files in the directory
+///
+/// # Panics
+/// It will panic if:
+/// - The provided path doesn't exist.
+/// - The process lacks permissions to view the contents.
+/// - The path points at a non-directory file.
+///
+/// # Example
+/// ```
+/// use utils::count_files;
+/// use utils::get_home_dir;
+///
+/// let path_to_keystore = get_home_dir() + "/.forest/libp2p/keypair";
+/// let file_count = count_files(path_to_keystore[0..path_to_keystore.len()-8].to_string());
```

Comment:
why are you indexing to length - 8?

Reply:
Removing the filename for now. I can replace it like "/keypair".len().